src/settings/LoanHistory/LoanHistoryForm.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-unused-state */
```

Comment:
I am more inclined to have eslint-disable comments in place rather than disabling the whole file.